EXTERNAL_IMAGES.md

```diff
@@ -42,4 +42,42 @@ export function decorateMain(main) {
 ## How does this work?
 During the page rendering process, the frontend code replaces the anchor tags identified as exteernal images on the page with the `picture` tags with `src`/`srcset` attributes set as the external image's url as specified in the external image link placed on the Word / Google Document during the page authoring process.
 
-Authors can optionally specify query paramaters in the hyperlinked external url and they would be retained in the `picture` tag's `src`/`srcset` attributes. These are useful for specifying image delivery parameters such as image width, height, format, etc. as understood by the external image delivery service.
\ No newline at end of file
+Authors can optionally specify query paramaters in the hyperlinked external url and they would be retained in the `picture` tag's `src`/`srcset` attributes. These are useful for specifying image delivery parameters such as image width, height, format, etc. as understood by the external image delivery service.
+
+## Extended Usecase of leveraging DM OpenAPI for Streamlining AEM Franklin Development
+
+For clients utilizing the `AEM Assets` repository to distribute their assets, the burden of maintaining asset integrity due to frequent edits, updates, and deletions performed within AEM is alleviated. Thanks to `Dynamic Media OpenAPI`, synchronization with every DOCX document becomes seamless.
+
+Simply incorporate the necessary `asset-library` configuration into AEM sidekick plugin, as outlined in [here](https://github.com/hlxsites/franklin-assets-selector/commit/24e99a2455c9372c8a54637f3e24f0fe4c2ac4f5)
+
+Once `asset-library` in integrated then simply perform copy/paste through plugin for any asset into target DOCX. Here is one crafted sample on how it might look like - [Check Here](https://ext-images-with-dm-openapi--franklinvideo--anuraggupta228.hlx.live/dm-open-api-with-aem-franklin)
+
+And voilà! It’s done! Not only have we minimized redundancy and alleviated the cumbersome task for clients to update every DOCX document with `Dynamic Media Open API` but the LHS still preserves at 100 for mobile as well as desktop - [Witness Here!](https://pagespeed.web.dev/analysis/https-dm-openapi-with-eds--franklin-assets-selector--hlxsites-hlx-live-anuraggupta228-DM-OpenAPI-adding-refs-in-eds/1qsej8ad01?form_factor=mobile)
+
+Now, let’s take a closer look at the comprehensive PSI report: ![Page Speed Insights Report](/images/psi-report-for-DM-OpenAPI.png)
```

Comment:
I think this needs thorough more analysis before being added.  Also its now possible to serve images from the site domain itself instead of a separate delivery domain by doing the asset delivery routing to Polaris origin at the CDN level. So the analysis should ideally be based on that setup.

Reply:
Removed it for now from this PR